media/tracker_modules/xm.ksy

```diff
@@ -0,0 +1,280 @@
+meta:
+  id: xm
+  title: Extended Module
+  application:
+    - FastTracker 2
+    - Protracker
+    - MilkyTracker
+    - libmodplug
+    - Mikmod
+    - Extended Module Player
+    - VLC
+    - Media Player Classic
+    - MPlayer
+  file-extension: xm
+  license: Unlicense
+  endian: le
+  encoding: utf-8
```

Comment:
Again, given that xm originated from FastTracker in early 1990s, I highly doubt that any utf-8 support existed back there at all.

Reply:
I guess it is quite safe to put utf-8 everywhere where text-ASCII is assumed (without special characters) because it is compatible with ASCII.

Reply:
As far as I see, there are tons of xm modules which make liberal use of non-ASCII names everywhere, usually assuming something like cp437 or some similar IBM codepage.

Reply:
I can make it an array then. But it is not very good idea - ksy descriptions are also a documentation, module name is meant to be a string, but we don't have strings of unknown encoding for now.
How about ```encpding: ?``` triggering heuristical encoding detection?